api-docs-slate/source/index.html.md

```diff
@@ -40,41 +40,57 @@ search: true
 
 # Introduction
 
-Welcome to the Bcoin API!
+Welcome to the bcoin API!
 
-The default bcoin HTTP server listens on the standard RPC port (`8332` for main, `18332` for testnet, `48332` for regtest, and `18556` default for simnet). It exposes a REST json api, as well as a JSON-RPC api.
+The default bcoin HTTP server listens on the standard RPC port (`8332` for main, `18332` for testnet, `48332` for regtest, and `18556` default for simnet). It exposes a REST JSON, as well as a JSON-RPC api.
 
 # Authentication
 ## Auth
 
 ```shell--curl
-curl http://x:[api-key]@127.0.0.1:8332/
+# port determines network type (48332 for regtest), API key required in URL
```

Comment:
This isn't necessarily true. As mentioned in another comment, you can set custom ports when starting up your node (I believe there is a `port` argument) so these are just the default, not necessarily what determines your network type.